lib/repos/settings_repository.dart

```diff
@@ -0,0 +1,7 @@
+class SettingsRepository {
+  Future<void> updateFullName(String fullName) async {}
```

Comment:
Maybe a good mock behavior could be Future.delayed(Duration(seconds: N,));

Reply:
Okay cool.